db/migrations/20200929-nonSSP-synthetic-data.sql

```diff
@@ -0,0 +1,185 @@
+INSERT INTO public.cclf_files (id, created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status) VALUES (1853, now(), now(), 8, 'T.CEC.ZC8Y20.D200929.T0000000', 'E9994', '2020-02-01', 20, 'Completed');
```

Comment:
Should we annotate this script to note that this was ran on sandbox and it's for tracking/auditing only?